### DISCUSSION TITLE
MantineProvider 7.1.5 can't use partially in single page app.

### DISCUSSION THREAD

**Discussion by jessemoe** [mantinedev/mantine#5095] [Q&A]
### What package has an issue

@mantine/core

### Describe the bug

`Uncaught Error: @mantine/core: MantineProvider was not found in component tree, make sure you have it in your app`
I don't want to use Mantine in my full app

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.1.5

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
MantineProvider must be in the component tree. It may be the entire application or a part of it.
